intro_community.qmd

```diff
@@ -15,15 +15,15 @@ Community members are people who use, cite and share use cases for rOpenSci pack
 attend or present in our Community Calls, write a post for our blog, 
 participated in an rOpenSci unconf, or in our research domain focused communities like rOpenSci-Antarctic and Southern Ocean Research, 
 answer questions in our fora, actively participate in the life of an rOpenSci project reporting problems and recommending or implementing solutions, 
-contribute to, or maintain a package, or participate in rOpenSci peer review as a reviewer, 
+contribute to, or maintain a package, being champions, mentor or trainer for our Champions Program, localize our material, or participate in rOpenSci peer review as a reviewer, 
 author or editor.
 
 Having a diverse community is important to us. 
 We welcome anyone dedicated to making their science more open and supporting others in that pursuit, 
 no matter their technical expertise, career stage, or work sector.
 Further, we welcome people of all backgrounds, including but not limited to any sexuality, 
 gender identity, and race. We are anti-racist. 
-We recognize that communities are not inclusive by default, particularly as they grow: they require deliberate attention. 
+We recognize that communities are not inclusive by default, particularly as they grow: they require deliberate attention. We are carrying out a Champions Program and a Localization project to ensure our research software serves everyone in our communities. 
```

Comment:
More than one localization projects? Because more than one language? Or does it depend?

```suggestion
We recognize that communities are not inclusive by default, particularly as they grow: they require deliberate attention. We are carrying out a Champions Program and Localization projects to ensure our research software serves everyone in our communities. 
```

Reply:
"Projects" is correct; now, we manage one project per language.